src/index.ts

```diff
@@ -37,6 +45,10 @@ export function shard<T>(
         seed: defaultSeed,
         throwOnEmpty: true,
         throwWhenNotSharding: true,
+        // we don't import this one, because it statically exposes the vars
+        // and we need to be able to mock them for tests
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        concurrency: require('ci-parallel-vars'),
```

Comment:
can we statically `import` this anyway, please? It is a cheap evaluation and is used for most places. Given that we make it part of the `defaultOptions` anyway means that we load and evaluate it no matter what.

Reply:
We can't because we use a mocked version for tests and need to be imported every time the function is being executed.